tests/postgres_tests/test_hstore.py

```diff
@@ -68,6 +70,7 @@ def test_array_field(self):
         self.assertEqual(instance.array_field, expected_value)
 
 
+@modify_settings(INSTALLED_APPS={"append": "django.contrib.postgres"})
```

Comment:
Added to avoid `can't adapt dict` where type handlers were not registered (follow up to 3b3f38b3b09b0f2373e51406ecb8c9c45d36aebc).

Reply:
I fixed this in my psycopg3 branch like this: https://github.com/django/django/pull/15687/commits/964075499137baecb82063a23c10f974a222250c 

Covering this for the whole class might make more sense.